### DISCUSSION TITLE
Casual Dired v1.0.4 Released!

### DISCUSSION THREAD

**Discussion by kickingvegas** [kickingvegas/casual-dired#33] [Announcements]
Casual Dired [v1.0.4](https://github.com/kickingvegas/casual-dired/releases/tag/1.0.4) released and now on [MELPA](https://melpa.org/#/casual-dired). Adds fix to support older versions of GNU coreutils `ls` (v8.32) so that sorting by modification date works.
Thanks to @dandrake for raising this issue and assistance in helping fix it.

**Comment by dandrake** [comment]
Two comments on the support for older versions of `ls` -- I stumbled on this from Windows, but my Raspberry Pi, running Raspbian 11, also uses that coreutils version 8.32. So the change will also make this package work for those using emacs on their Raspberry Pis.

Second, I made a show-and-tell discussion -- #37 -- detailing how I stumbled around to find this and make it work on Windows.

Thanks for the update. This package really makes dired more usable for me!

**Reply by kickingvegas** [reply]
@dandrake From another question you posted related to cc-isearch-menu, with v1.1.0 there is now support for I-search across multiple files. https://github.com/kickingvegas/cc-isearch-menu/issues/8
